src/commands/database-import.ts

```diff
@@ -0,0 +1,96 @@
+import * as clc from "colorette";
+import * as fs from "fs";
+import * as utils from "../utils";
+
+import { Command } from "../command";
+import DatabaseImporter from "../database/import";
+import { Emulators } from "../emulator/types";
+import { FirebaseError } from "../error";
+import { logger } from "../logger";
+import { printNoticeIfEmulated } from "../emulator/commandUtils";
+import { promptOnce } from "../prompt";
+import { populateInstanceDetails } from "../management/database";
+import { realtimeOriginOrEmulatorOrCustomUrl } from "../database/api";
+import { requireDatabaseInstance } from "../requireDatabaseInstance";
+import { requirePermissions } from "../requirePermissions";
+
+export const command = new Command("database:import <path> [infile]")
+  .description("non-atomically import JSON file to the specified path")
+  .option("-f, --force", "pass this option to bypass confirmation prompt")
+  .option(
+    "--instance <instance>",
+    "use the database <instance>.firebaseio.com (if omitted, use default database instance)"
+  )
+  .option(
+    "--disable-triggers",
+    "suppress any Cloud functions triggered by this operation, default to true",
+    true
+  )
+  .option(
+    "--only <dataPath>",
+    "import only data at this path in the JSON file (if omitted, import entire file)"
+  )
+  .before(requirePermissions, ["firebasedatabase.instances.update"])
+  .before(requireDatabaseInstance)
+  .before(populateInstanceDetails)
+  .before(printNoticeIfEmulated, Emulators.DATABASE)
+  .action(async (path: string, infile, options) => {
```

Comment:
Please add a type for infile and options. 

For options, I suggest extending the type defined in options.ts with any command  specific fields (like we do for hosting in hosting/options.ts)

This will fix the 'unsafe member access' linter issues that are all over this file.